webhook_server/utils/github_retry.py

```
@@ -0,0 +1,128 @@
+
+def _is_retryable(ex: Exception) -> bool:
+    """Determine whether an exception is retryable."""
+    if isinstance(ex, (BadCredentialsException, UnknownObjectException)):
+        return False
+
+    if isinstance(ex, GithubException):
+        if ex.status in _PERMANENT_STATUS_CODES:
+            return False
+        if ex.status in _RETRYABLE_STATUS_CODES:
+            return True
+        return False
```

Ruff (0.15.9)

[warning] Return the condition `ex.status in _RETRYABLE_STATUS_CODES` directly

Replace with `return ex.status in _RETRYABLE_STATUS_CODES`

(SIM103)